### PR TITLE
Working CA

### PR DESCRIPTION
**Pull request by Jkohl1**

### CODE REVIEW

File: Class-Code/Elementary-CA/jkohl/sketch.js [hunk 1]
```diff
@@ -1,14 +1,52 @@
-//Your Name(s)
```
Comment (pryan4) [inline Class-Code/Elementary-CA/jkohl/sketch.js:1]: need header

File: Class-Code/Elementary-CA/jkohl/sketch.js [hunk 2]
```diff
@@ -1,14 +1,52 @@
...
+    for(let j = 0; j < ca.length; j++)
+        for(let i = 0; i < gensize; i++){
+            if(ca[j][i] == 1){
+                rect(i*gridsize,j*gridsize,gridsize,gridsize)
+        }
+    }
+
+
+
+    //generate
```
Comment (pryan4) [inline Class-Code/Elementary-CA/jkohl/sketch.js:32]: don't need these

File: Class-Code/Elementary-CA/jkohl/sketch.js [hunk 3]
```diff
@@ -1,14 +1,52 @@
...
+    gen0[floor(gensize/2)]=1;
+    ca.push(gen0);
+    gridsize = width/gensize
 }
 
-function draw() {
-    background(220);
+function draw(){
+
+    //display
```
Comment (pryan4) [inline Class-Code/Elementary-CA/jkohl/sketch.js:20]: remove

File: Class-Code/Elementary-CA/jkohl/sketch.js [hunk 4]
```diff
@@ -1,14 +1,52 @@
...
+    let gen0 = Array(gensize).fill(0);
+    gen0[floor(gensize/2)]=1;
+    ca.push(gen0);
+    gridsize = width/gensize
 }
 
-function draw() {
-    background(220);
+function draw(){
+
```
Comment (pryan4) [inline Class-Code/Elementary-CA/jkohl/sketch.js:19]: need a background command

File: Class-Code/Elementary-CA/jkohl/sketch.js [hunk 5]
```diff
@@ -1,14 +1,52 @@
-//Your Name(s)
-//Project Name
-//Date
-
-/* A brief description of what the program is */
-
```
Comment (Mr-Darfler) [inline Class-Code/Elementary-CA/jkohl/sketch.js]: Make sure to include:
- Name
- Project Name
- Date
- A brief description of the code

File: Class-Code/Elementary-CA/jkohl/sketch.js (hunk 3 above)
Comment (Mr-Darfler) [reply Class-Code/Elementary-CA/jkohl/sketch.js:20]: If you're not going to turn all of this into a function, I would leave in the comment

File: Class-Code/Elementary-CA/jkohl/sketch.js (hunk 2 above)
Comment (Mr-Darfler) [reply Class-Code/Elementary-CA/jkohl/sketch.js:32]: Again, I think leaving the comment does help structure the code

**Review by Mr-Darfler — CHANGES_REQUESTED** [review]
You're pretty close @Jkohl1 Make sure that you add the header info

**Review by fcsmakers — APPROVED** [review]

**Review by Mr-Darfler — APPROVED** [review]

**Pull request merged**